packages/now-next/test/fixtures/06-lambda-with-memory/src/pages/api/index.js

```diff
@@ -0,0 +1,3 @@
+export default function (req, res) {
+  res.end(`${process.env.AWS_LAMBDA_FUNCTION_MEMORY_SIZE}`);
```

Comment:
You don't really need the backticks here I think, it's always going to be a string (or undefined, but that's ok for `res.end()` as well).

```suggestion
  res.end(process.env.AWS_LAMBDA_FUNCTION_MEMORY_SIZE);
```